spec/PeridicBackgroundSync-index.bs

```diff
@@ -0,0 +1,140 @@
+<pre class='metadata'>
+Title: Web Periodic Background Synchronization
+Status: CG-DRAFT
+ED: https://wicg.github.io/BackgroundSync/spec/
+Shortname: periodic-background-sync
+Level: 1
+Editor: Mugdha Lakhani, Google, nator@chromium.org
+Abstract: This specification describes a method that enables web applications to periodically synchronize data and content in the background.
+Group: wicg
+Repository: WICG/BackgroundSync
+Link Defaults: html (dfn) allowed to show a popup/event handler idl attribute/global object/in parallel/incumbent settings object/perform a microtask checkpoint/queue a task/script execution environment
+</pre>
+
+<pre class=biblio>
```

Comment:
None of the links in lines 14->81 are being sued

Reply:
yes, I got a warning about them. I was planning to use them later which is why I'd left them here. I'll remove and re-add when I'm using them.

Reply:
Turns out I'm using a few of these -- removing them causes ambiguity in the definitions of some terms used in the text. I've removed all of the ones not currently being used.